src/client/styles/components/_VotingBox.scss

```diff
@@ -0,0 +1,30 @@
+.voting-box {
+  position: absolute;
+  right: 0px;
+  margin-right: 100px;
+  z-index: 3;
+  height: 40px;
+  padding: 0 10px;
+  background-color: $bg-color;
+  outline: none;
+  border: 2px solid lighten($gray4, 20%);
+  border-radius: 10px;
+  color: lighten($gray4, 20%);
+  font-size: 1.2em;
```

Comment:
consider using px instead of em's to make it consistent across all the project
Em's are hard to predict because they depend on parent fonts size. 
In my opinion, if I would use relative units I'll always use REM, cause they're much more predictable